projects/flora/api/src/DataFixtures/PersonApiKeyFixtures.php

```
@@ -0,0 +1,43 @@
+
+use App\Entity\Person;
+use App\Entity\PersonApiKey;
+use App\Factory\PersonApiKeyFactory;
+use App\Security\Constant\PersonPermission;
+use Doctrine\Bundle\FixturesBundle\Fixture;
+use Doctrine\Common\DataFixtures\DependentFixtureInterface;
+use Doctrine\Persistence\ObjectManager;
+
+class PersonApiKeyFixtures extends Fixture implements DependentFixtureInterface
+{
+    public function load(ObjectManager $manager): void
```

PHPMD (2.15.0)

Avoid unused parameters such as '$manager'. (Unused Code Rules)

(UnusedFormalParameter)